modules/lambda-kms/templates/kms_key_policy.tpl

```diff
@@ -0,0 +1,36 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "Allow administration of the key",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": [
+          "arn:aws:iam::${aws_account_id}:root",
+          "${aws_user_arn}"
+        ]
+      },
+      "Action": [
+        "kms:*"
```

Comment:
I think it might be worthwhile to allow `root` and the deployment user access to `kms:*`, but also add a `deny` on `decrypt` (and possibly any other sensitive functions). This will allow the user to administer the key for deployment purposes, but not open up encrypted data to a wider audience than necessary.

Reply:
Decrypt is needed for `fleece config`.